deploy/overlays/helm/kustomization.yaml

```
@@ -1,14 +1,20 @@
 # This is the cluster wide security-profiles-operator deployment, which listens for
 # configMaps on all namespaces
----
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-bases:
-  - ../../base
 
-patchesJson6902:
-  - target:
-      version: v1
-      kind: ConfigMap
-      name: security-profiles-operator-profile
-    path: operator-profile.yaml
+resources:
+- ../../base
+patches:
+- path: operator-profile.yaml
+  target:
+    name: security-profiles-operator*
+- path: operator-profile.yaml
+  target:
+    name: spo*
+- path: operator-profile.yaml
+  target:
+    name: default
+- path: operator-profile.yaml
+  target:
+    name: metrics*
```

Let's do this by type instead:

```suggestion
patches:
- path: operator-profile.yaml
  target:
    kind: (ServiceAccount|Role|ClusterRoleBinding|RoleBinding|Secret)

- patch: |
    - op: replace
      path: /subjects/0/namespace
      value: "{{ .Release.Namespace }}"
  target:
    kind: (ClusterRoleBinding|RoleBinding)

- patch: |
    - op: replace
      path: /metadata/name
      value: "{{ .Release.Namespace }}"
  target:
    kind: Namespace
```